python/ray/data/_internal/logical/operators/map_operator.py

```diff
@@ -26,8 +26,29 @@
 class AbstractMap(LogicalOperator):
     """Abstract class for logical operators should be converted to physical
     MapOperator.
+
+    Args:
+        name: Name for this operator. This is the name that will appear when inspecting
+            the logical plan of a Dataset.
+        input_op: The operator preceding this operator in the plan DAG. The outputs of
+            `input_op` will be the inputs to this operator.
+        block_fn: The transform function to apply to each input block to produce output
+            blocks.
+        target_block_size: The target size for blocks outputted by this operator.
+        fn: User provided UDF. `fn` is used as the first argument to `block_fn` if
+            provided.
```

Comment:
> `fn` is used as the first argument to `block_fn` if provided.

hmm not sure if I understand, but this is not what we are doing right? Can we say something like:  `fn` is called inside `block_fn`.

Reply:
this is what's happening in `compute.py`

Reply:
oh it's actually second argument - https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/compute.py#L483 . But I don't think we need to give this level of detail on docstring, right?

Reply:
sg, removed